source/ShipyardPanel.cpp

```diff
@@ -120,8 +120,11 @@ int ShipyardPanel::DrawPlayerShipInfo(const Point &point)
 {
 	shipInfo.Update(*playerShip, player, collapsed.count("description"));
 	shipInfo.DrawAttributes(point, true);
+	const int attributesHeight = shipInfo.GetAttributesHeight(true);
+	const Point outfitsPoint(point.X(), point.Y() + attributesHeight);
+	shipInfo.DrawOutfits(outfitsPoint);
```

Comment:
```suggestion
	shipInfo.DrawOutfits({point.X(), point.Y() + attributesHeight});
```
not sure if I did this right, not doing this often, alternative:
```suggestion
	shipInfo.DrawOutfits(Point(point.X(), point.Y() + attributesHeight));
```

same thing here